EpamTests/StepDefinitions/EpamDownloadSteps.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using EpamTests.Pages;
+using EpamTests.Tests;
+using EpamTests.Utils;
+using TechTalk.SpecFlow;
+
+namespace EpamTests.StepDefinitions
+{
+    [Binding]
+    public class EpamDownloadSteps : BaseTest
+    {
+        private string? downloadedFilePath;
```

Comment:
u dont need this field.
create local variable where u need
var downloadedFilePath = Path.Combine(Constants.FilePaths.DownloadsPath, fileName);